tsl/test/sql/data/copy_data.csv

```diff
@@ -0,0 +1,1002 @@
+generate_series,random
```

Comment:
Is possible to use a small dataset?

Reply:
Not really, it needs at least 1001 rows in order to trigger this because of the sizes of copy buffers.